README.md

```diff
@@ -37,9 +38,11 @@ pip install -r requirements.txt
 * Request an API key from [Event Registry](https://www.eventregistry.org)
 * Request an API key for the [Google Civic Information API](https://developers.google.com/civic-information)
 * [Optional] Request credentials from [Twilio](https://www.twilio.com/) and a phone number and [application sid](https://www.twilio.com/console/voice/dev-tools/twiml-apps):
-* Add the following to a file:
+* Add the following to a file (either name it `indivisible.cfg`, or modify the path in the
+  instructions below):
 ~~~
 PYTHONPATH=/path/to/repo/indivisible
+FLASK_DEBUG=1
```

Comment:
Seems easier to specify it here than to have to remember to `export` it every time you open a new terminal session, but let me know if you disagree.